wasm/jit/jit_arm64.go

```diff
@@ -581,20 +681,152 @@ func (c *arm64Compiler) compileEqz(o *wazeroir.OperationEqz) error {
 	return fmt.Errorf("TODO: unsupported on arm64")
 }
 
+// compileLt implements compiler.compileLt for the arm64 architecture.
 func (c *arm64Compiler) compileLt(o *wazeroir.OperationLt) error {
-	return fmt.Errorf("TODO: unsupported on arm64")
+	x1, x2, err := c.popTwoValuesOnRegisters()
+	if err != nil {
+		return err
+	}
+
+	var inst obj.As
+	var conditionalRegister conditionalRegisterState
+	switch o.Type {
+	case wazeroir.SignedTypeUint32:
+		inst = arm64.ACMPW
+		conditionalRegister = arm64.COND_LO
+	case wazeroir.SignedTypeUint64:
+		inst = arm64.ACMP
+		conditionalRegister = arm64.COND_LO
+	case wazeroir.SignedTypeInt32:
+		inst = arm64.ACMPW
+		conditionalRegister = arm64.COND_LT
+	case wazeroir.SignedTypeInt64:
+		inst = arm64.ACMP
+		conditionalRegister = arm64.COND_LT
+	case wazeroir.SignedTypeFloat32:
+		inst = arm64.AFCMPS
+		conditionalRegister = arm64.COND_MI
+	case wazeroir.SignedTypeFloat64:
+		inst = arm64.AFCMPD
+		conditionalRegister = arm64.COND_MI
+	}
+
+	c.applyTwoRegistersToNoneInstruction(inst, x2.register, x1.register)
+
+	// Push the comparison result as a conditional register placed value.
```

Comment:
"conditional register placed value" is a mouthful and makes me wonder if the word "placed" is meaningful. If it isn't, delete it, or perhaps move the significance of "placed" terminology to doc on `pushValueOnConditionalRegister` and leave it out here. similar comment for all "placed" later in the file